README.md

```diff
@@ -203,51 +205,65 @@ Open the project in your IDE to configure the code.
 - Note the context-sensitive button now says `Sign out` and displays your username to its left.
 - The middle of the screen now has an option to click for ID Token Details: click it to see some of the ID token's decoded claims.
 - You can also use the button on the top right to sign out.
-- After signing out, click this link to the [token details page](https://127.0.0.1:5000/auth/token_details) to observe how the app displays a `401: unauthorized` error instead of the ID token claims.
+- After signing out, click this link to the [token details page](https://127.0.0.1:5000/auth/token_details) to observe that the app displays a `401: unauthorized` error instead of the ID token claims when the user is not authorized.
 
 > :information_source: Did the sample not work for you as expected? Did you encounter issues trying this sample? Then please reach out to us using the [GitHub Issues](../issues) page.
 
 ## About the code
 
-This sample shows how to use [Microsoft Authentication Library \(MSAL\) for Python](https://github.com/AzureAD/microsoft-authentication-library-for-python) to sign in users from your Azure AD tenant. 
+This sample uses the [Microsoft Authentication Library \(MSAL\) for Python](https://github.com/AzureAD/microsoft-authentication-library-for-python) to sign in users within your Azure AD tenant. It levarages the IdentityWebPython class found in the [Microsoft Identity Python Samples Common](https://github.com/azure-samples/ms-identity-python-common) repository to allow for quick app setup.
 
-* A **ConfidentialClientApplication** object is instantiated in the [auth_endpoints.py](auth_endpoints.py) file. The following parameters need to be provided upon instantiation:
+1. A configuration object is parsed from [aad.config.json](./aad.config.json)
```

Comment:
>1. [](start = 0, length = 2)

I'd start this section with "In app.py, method def create_app(secure_client_credential=None):"